client/signup/jetpack-connect/site-url-input.jsx

```diff
@@ -22,8 +22,9 @@ export default React.createClass( {
 	},
 
 	onChange( event ) {
+		const url = event.target.value;
 		this.setState( {
-			value: event.target.value
+			value: url.replace(/\/$/, "")
```

Comment:
(I must admit that being able to read the method as "untrailings-la-shit" makes want to use it )